packages/nexus-sdk/src/Archive/index.ts

```diff
@@ -0,0 +1,43 @@
+import { Fetchers, Resource } from '../types';
+import { NexusContext } from '../nexusSdk';
+import { ArchivePayload } from './types';
+
+const Archive = (
+  { httpGet, httpPut, httpPost, poll }: Fetchers,
+  context: NexusContext,
+) => {
+  return {
+    get: <T>(
```

Comment:
According to the doc, there are a couple of ways you can retrieve an archive (content or metadata): https://bluebrainnexus.io/docs/api/current/kg/kg-archives-api.html#fetch-an-archive

I would add options, like we did for Resource https://github.com/BlueBrain/nexus-js/blob/master/packages/nexus-sdk/src/Resource/types.ts#L53 and https://github.com/BlueBrain/nexus-js/blob/master/packages/nexus-sdk/src/Resource/index.ts#L20-L26 with maybe `x-tar` being the default?